config/400-default-ch-config.yaml

```diff
@@ -0,0 +1,29 @@
+# Copyright 2018 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: default-ch-webhook
+  namespace: knative-eventing
+data:
+  # Configuration for defaulting channels that do not specify CRD implementations.
+  default-ch-config: |
+    clusterdefault:
+      apiVersion: messaging.knative.dev/v1alpha1
+      kind: InMemoryChannel
```

Comment:
Should there be a way to specify specs as well?

Reply:
... time passes ... looks like the config is indeed a channeltemplatespec which includes the spec portion of it as well. We can probably handle this via abovementioned documentation for the usage with some examples.

Reply:
Yes, that's correct. With a few examples in a follow up documentation PR I think we should be fine.
Thanks!